tables.sql

```diff
@@ -0,0 +1,87 @@
+CREATE DATABASE thebackups_TaylorC;
+
+USE thebackups_TaylorC;
+
+CREATE TABLE employee_type (
+	`employee_type_id` TINYINT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    `name` VARCHAR(30) NOT NULL
+);
+
+CREATE TABLE employee (
+	`employee_id` SMALLINT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    `first_name` VARCHAR(35) NOT NULL,
+    `last_name` VARCHAR(35) NOT NULL,
+    `salary` MEDIUMINT UNSIGNED NOT NULL,
+    `sort_code` VARCHAR(6) NOT NULL,
+    `account_number` VARCHAR(8) NOT NULL,
+    `nin` VARCHAR(9) NOT NULL,
+    `commission` DECIMAL(10, 2) NULL,
+    `username` VARCHAR(35) NOT NULL,
+    `password` VARCHAR(35) NOT NULL,
+    `type_id` TINYINT UNSIGNED NOT NULL,
+    CHECK (LENGTH(sort_code) = 6),
+    CHECK (LENGTH(account_number) = 8),
+    CHECK (LENGTH(nin) = 9),
+    CHECK (salary > 0 AND salary <= 500000)
+);
+
+ALTER TABLE employee ADD CONSTRAINT fk_employee_employee_type FOREIGN KEY(type_id) REFERENCES employee_type(employee_type_id);
+
+CREATE TABLE `client` (
+    `client_id` SMALLINT UNSIGNED PRIMARY KEY,
+    `name` VARCHAR(255) NOT NULL,
+    `address` VARCHAR(255) NOT NULL,
+    `phone` VARCHAR(11) NOT NULL,
+    `employee_id` SMALLINT UNSIGNED NOT NULL,
+    `win_date` TIMESTAMP NULL
+);
+
+ALTER TABLE `client` ADD CONSTRAINT fk_client_employee FOREIGN KEY(employee_id) REFERENCES employee(employee_id);
```

Comment:
The constraint could be added when creating the table

Reply:
All Fixed!